builder/files/etc/rc.local

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
```

Comment:
And one more spot I'm uncertain - this doesn't exist in https://github.com/publiclab/pi-builder/pull/40/files for example, which builds properly and is supposed to be almost the same as this PR.